Interpreter/Html/Form.php

```diff
@@ -301,14 +301,25 @@ public function isValid($revalid = false)
             return $this->_validity = false;
         }
 
-        $elements   = $this->getElements();
-        $names      = [];
-        $validation = [];
-
+        $elements     = $this->getElements();
+        $names        = [];
+        $validation   = [];
+        $nameIterator = [];
```

Comment:
Insert a new line after this instruction. `foreach` block must surrounded by empty line :-).